src/full_beaconchain/full_beaconchain_launcher.star

```diff
@@ -0,0 +1,255 @@
+shared_utils = import_module("github.com/kurtosis-tech/eth2-package/src/shared_utils/shared_utils.star")
+IMAGE_NAME = "gobitfly/eth2-beaconchain-explorer:kurtosis"
+
+POSTGRES_PORT_ID = "postgres"
+POSTGRES_PORT_NUMBER = 5432
+POSTGRES_DB = "db"
+POSTGRES_USER = "postgres"
+POSTGRES_PASSWORD = "pass"
+
+REDIS_PORT_ID = "redis"
+REDIS_PORT_NUMBER = 6379
+
+FRONTEND_PORT_ID     = "http"
+FRONTEND_PORT_NUMBER = 8080
+
+LITTLE_BIGTABLE_PORT_ID = "littlebigtable"
+LITTLE_BIGTABLE_PORT_NUMBER = 9000
+
+FULL_BEACONCHAIN_CONFIG_FILENAME = "config.yml"
+
+
+USED_PORTS = {
+    FRONTEND_PORT_ID:shared_utils.new_port_spec(FRONTEND_PORT_NUMBER, shared_utils.TCP_PROTOCOL, shared_utils.HTTP_APPLICATION_PROTOCOL)
+}
+
+def launch_full_beacon(
+    plan,
+        config_template,
+        cl_client_contexts,
+        el_client_contexts,
+    ):
+
+    db_services = plan.add_services(
+        configs={
+	        # Add a Postgres server
+            "explorer-postgres": ServiceConfig(
```

Comment:
can we use the https://github.com/kurtosis-tech/redis-package and https://github.com/kurtosis-tech/postgres-package

for this?